datafeeder/src/main/java/org/georchestra/datafeeder/service/publish/impl/GeorchestraDataBackendService.java

```diff
@@ -181,7 +181,14 @@ private String resolveTargetTypeName(@NonNull DatasetUploadState dataset, Map<St
             throw new IllegalStateException("Georchestra organization name not provided in job.user.organization.id");
         }
         String schema = nameResolver.resolveDatabaseSchemaName(orgName);
-        connectionParams.put(PostgisNGDataStoreFactory.SCHEMA.key, schema);
+
+        for (String k : connectionParams.keySet()) {
+            String v = connectionParams.get(k);
+            if ("<schema>".equals(v)) {
+                connectionParams.put(PostgisNGDataStoreFactory.SCHEMA.key, schema);
+            }
+        }
```

Comment:
so it means that `connectionParams` already contains a `schema` entry ? I don't think it is necessary to iterate over every parameters though. Maybe something like:
```
def schema = connectionParams.get(PostgisNGDataStoreFactory.SCHEMA.key);
if (schema.equals("<schema>") {
  connectionParams.put(PostgisNGDataStoreFactory.SCHEMA.key, schema);
} else {
 // maybe still the need to normalize the field ? https://github.com/georchestra/georchestra/blob/24cd6aa3b245860a8f5a133e2802577af377a68c/datafeeder/src/main/java/org/georchestra/datafeeder/autoconf/GeorchestraNameNormalizer.java#L73-L95
}
```

Reply:
I did it like that to be aligned with this that do the same : https://github.com/georchestra/georchestra/blob/master/datafeeder/src/main/java/org/georchestra/datafeeder/service/publish/impl/GeorchestraOwsPublicationService.java#L213